akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -0,0 +1,54 @@
+###############################################
+# Akka Cluster Sharding Reference Config File #
+###############################################
+
+# This is the reference config file that contains all the default settings.
+# Make your edits/overrides in your application.conf.
+
+
+# //#sharding-ext-config
+# Settings for the ClusterShardingExtension
+akka.cluster.sharding {
+  # The extension creates a top level actor with this name in top level user scope,
+  # e.g. '/user/sharding'
+  guardian-name = sharding
```

Comment:
empty lines after each setting make this more readable where no syntax highlighting is available